web-animations-2/Overview.bs

```diff
@@ -3028,10 +3049,40 @@ with which it is associated is updated.
 
 </ins>
 
-<h3 id="the-animatable-interface">The <code>Animatable</code> interface</h3>
+The <code>Animatable</code> interface mixin {#the-animatable-interface-mixin}
+----------------------------------------
 
-<div class="methods">
+Objects that may be the target of an {{KeyframeEffect}} object implement
+the {{Animatable}} interface mixin.
+
+<pre class='idl'>
+interface mixin Animatable {
+    Animation           animate(object? keyframes,
+                                optional (unrestricted double or KeyframeAnimationOptions) options = {});
+    sequence&lt;Animation&gt; getAnimations(optional GetAnimationsOptions options = {});
+};
+
+dictionary TimelineRangeOffset {
+  CSSOMString? rangeName;
+  CSSNumericValue offset;
+};
+
+dictionary KeyframeAnimationOptions : KeyframeEffectOptions {
+    DOMString id = "";
+    AnimationTimeline? timeline;
+
+    (TimelineRangeOffset or CSSNumericValue or DOMString) rangeStart = "normal";
+    (TimelineRangeOffset or CSSNumericValue or DOMString) rangeEnd = "normal";
+};
 
+dictionary GetAnimationsOptions {
+    boolean subtree = false;
+};
+</pre>
+
+ISSUE: Should rangeStart/rangeEnd also accept CSSNumericValue directly (for cases that don't want a rangeName)?
```

Comment:
I'm not sure what this refers to. It looks like the definitions of `rangeStart` / `rangeEnd` already to accept a `CSSNumericValue`?

Reply:
Thanks! Removed; that was leftover from a previous version. ^_^;;